### PR TITLE
fix(gui): make sure customer name is a string

### PR DESCRIPTION
**Pull request by william-gr**
Ticket:	#25489

### CODE REVIEW

File: gui/support/views.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def ticket(request):
 
         license, reason = utils.get_license()
         if license:
-            company = license.customer_name
+            company = license.customer_name.decode()
         else:
```
Comment (surajrav) [inline gui/support/views.py]: I found another occurrence of this in a template: https://github.com/freenas/freenas/blob/freenas/11.0-stable/gui/templates/support/home.html#L58.

Should we take care of that too?

Comment (surajrav) [inline gui/support/views.py]: This (the support home.html template occurrence) also applies to the master branch as well, however in the master branch, this current views.py part is not even present. I suppose this is intentional and some other way was found in master to deal with/get to this information.

**Comment by vaibhav-rbs** [comment]
there are 2 checks which are failing, please fix or explain.

**Comment by surajrav** [comment]
@vaibhav-rbs that's just py-libzfs failing (I spoke to you about it)

**Comment by surajrav** [comment]
retest this please ( I just *attempted* another fix to py-libzfs, gah I'm not going well with this, it should have been a trivial thing to fix but here we are at my second attempt to fix it, oy vey)

**Comment by surajrav** [comment]
retest this please

**Pull request merged**